### ISSUE TITLE
Diagnostics are trimmed

### ISSUE THREAD

**Issue by arbitrary-dev** [neoclide/coc.nvim#1722]
## Result from CocInfo

```
## versions

vim version: VIM - Vi IMproved 8.1 (2018 May 18, compiled Mar  4 2020 10:55:18)
node version: v12.16.1
coc.nvim version: 0.0.77-817efd5108
term: dumb
platform: linux
```

## Describe the bug

Diagnostics are trimmed:
```
[bloop] [E] type mismatch;
 found    : com.example.ClassA ...
```

No full message in `:lopen` either.

## Expected

```
[bloop] [E] type mismatch;
 found    : com.example.ClassA
 expected : com.example.ClassB
```

…or at least a way to see full messages.

**Comment by fannheyward** [comment]
What language server do you use to reproduce this issue?

**Comment by arbitrary-dev** [comment]
[Metals](http://scalameta.org/metals) + [coc-metals](https://github.com/scalameta/coc-metals)

**Comment by chemzqm** [comment]
Please provide verbose output from language server, checkout https://github.com/neoclide/coc.nvim/wiki/Debug-language-server#using-output-channel

**Comment by fannheyward** [comment]
@ckipp01 can you take a look at this?

**Comment by ckipp01** [comment]
Sure, let me look into it.

**Comment by ckipp01** [comment]
So I'm not fully sure why it looks like it's getting truncated @arbitrary-dev. I tried to mimic exactly what you are showing, and this is what I get.

<img width="418" alt="Screenshot 2020-04-16 at 08 47 13" src="https://user-images.githubusercontent.com/13974112/79423465-f3dbbf00-7fbe-11ea-8387-9a8ab35ec36c.png">

Also when I look at the trace file, I see the full diagnostic message.

```json
[Trace - 08:46:28 AM] Sending notification 'textDocument/publishDiagnostics'
Params: {
  "uri": "file:///Users/ckipp/Documents/scala-workspace/test-project/example/src/Main.scala",
  "diagnostics": [
    {
      "range": {
        "start": {
          "line": 12,
          "character": 4
        },
        "end": {
          "line": 12,
          "character": 5
        }
      },
      "severity": 1,
      "source": "bloop",
      "message": "type mismatch;\n found   : example.Example.ClassB.type\n required: example.Example.ClassA"
    }
  ]
}
```
I tried this both in Vim and Nvim. I don't believe that we truncate any diagnostics coming to the user. One way to ensure this is to create a trace file, [we have instructions here](https://scalameta.org/metals/docs/contributors/getting-started.html#json-rpc-trace), and then look at the diagnostics coming from Metals/bloop. I don't believe it will be truncated, but that way you'll for sure know if you are getting them truncated, or if something with your vim setup or coc.nvim is truncating them.
